### ISSUE TITLE
[Startup: Launch Size]: Screen reader does not announce the sub text defined with 'Launch size' and 'Launch parameters' buttons.

### ISSUE THREAD

**Issue by Shubham786786** [microsoft/terminal#14400]
### Windows Terminal version

1.17.3052.0

### Windows build number

Windows 11 Version 22H2 (OS Build 25206.1000)

### Steps to reproduce

1. Open Windows Terminal.
2. Open 'Settings > Startup' window with 'Ctrl+,' keys.
3. Turn on Narrator by 'Ctrl+Win+Enter' keys.
4. Navigate to 'Launch size' button by tab key.
5. Observe the issue.

**User Experience: -**
Screen reader users will have to navigate in scan mode to get the subtitle text information defined with the button's text.

**Guideline Reference:**
(https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

**Attachments:**
[Screen reader does not announce the sub text defined with 'Launch size' and 'Launch parameters' buttons..zip](https://github.com/microsoft/terminal/files/10020773/Screen.reader.does.not.announce.the.sub.text.defined.with.Launch.size.and.Launch.parameters.buttons.zip)

### Expected Behavior

Screen reader should announce the sub text defined with 'Launch size' and 'Launch parameters' buttons. 

It should announce as '_Launch size group collapsed, the number of rows and columns displayed..._'.
and also, when navigating in scan mode, focus should move to this sub text which is not happening currently.

### Actual Behavior

Screen reader does not announce the sub text defined with 'Launch size' and 'Launch parameters' buttons. It announces as '_Launch size group collapsed_'.

**Observation:**
1. Issue also reproes with the NVDA screen reader.
2. This happens throughout the application for similar buttons.

**Comment by carlos-zamora** [comment]
@Shubham786786 I investigated this, and I'm a bit confused about my findings.

The control has the Full Description Automation Property defined like with the other controls in the settings UI. This was verified using Accessibility Insights. However, Narrator doesn't automatically read the full description as it does with other controls.

I don't think that means the bug is on us because we're setting the property. I also don't think that means the bug is on WinUI because the properties are being set and exposed properly.

Additionally, I verified that this does not repro using NVDA 2023.1.0.26827. You said above that the issue repros with NVDA. Which version are you using?

Since the properties are being set and exposed properly, and the bug does not occur in NVDA, I think this is actually a bug on Narrator and should be forwarded to them.

**Comment by zadjii-msft** [comment]
@Shubham786786 @v-rpundir can either of you comment on this re: Carlos's findings?

**Comment by Shubham786786** [comment]
I have verified the issue on the latest NVDA version, and it works fine, the issue is with Narrator, will forward the bug to Narrator team.
